### PR TITLE
New Themed Send Scene

### PR DESCRIPTION
**Pull request by amanalili**
This are the complete tested scenarios and results:
- Normal sending tested - Transaction Successful
- Send Scene selecting FIO address - Transaction Successful. Note: Memo not saved on Transaction Details.
- Create Wallet (EOS) then Send Scene - Transaction Failure. No Available Connections. Also failing on `develop` branch
- FIO Request List then Send Scene - Transaction Successful
- FIO Name Register then Send Scene - Transaction Successful
- FIO Custom Domain Register then Send Scene - Did not test, expensive domains.
- Sell thru gift cards bitrefill (Edge Provider) - Transaction Failure. No Available Connections. Also failing on `develop` branch

Note to reviewers: This feature has been progressively made so any comments on the earlier commits might have been fixed on the later commits.

#### PR Requirements
If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS Tablet
- [ ] Tested on small Android

New Send Scene:
https://user-images.githubusercontent.com/17136567/107926622-e8776c80-6fb0-11eb-98a7-9ff116dd44b9.mov

New Send Scene using FIO address
https://user-images.githubusercontent.com/17136567/107927055-73586700-6fb1-11eb-9e8a-a032a09d6896.mov

### CODE REVIEW

File: src/util/utils.js [hunk 1]
```diff
@@ -658,3 +660,90 @@ export function maxPrimaryCurrencyConversionDecimals(primaryPrecision: number, p
   const newPrimaryPrecision = primaryPrecision - precisionAdjustValue
   return newPrimaryPrecision >= 0 ? newPrimaryPrecision : 0
 }
+
+// Fees calculation
+export const calculateCryptoNetworkFee = (networkFee: string, displayMultiplier: string, exchangeMultiplier: string): string => {
```
Comment (amanalili) [inline src/util/utils.js:665]: convertToCryptoFee

File: src/util/utils.js [hunk 2]
```diff
@@ -658,3 +660,90 @@ export function maxPrimaryCurrencyConversionDecimals(primaryPrecision: number, p
...
+  const exchangeToDisplayMultiplierRatio = bns.div(exchangeMultiplier, displayMultiplier, DIVIDE_PRECISION)
+  return bns.mul(cryptoFeeExchangeDenomAmount, exchangeToDisplayMultiplierRatio)
+}
+
+export const feeStyle = {
+  danger: 'dangerText',
+  warning: 'warningText'
+}
+
+export const calculateFiatNetworkFee = (
```
Comment (amanalili) [inline src/util/utils.js:676]: convertToFiatFee

File: src/util/utils.js [hunk 3]
```diff
@@ -658,3 +660,90 @@ export function maxPrimaryCurrencyConversionDecimals(primaryPrecision: number, p
...
+  const cryptoFeeExchangeAmount = convertNativeToExchange(exchangeMultiplier)(networkFee)
+  const fiatFeeAmount = convertCurrencyFromExchangeRates(exchangeRates, currencyCode, isoFiatCurrencyCode, parseFloat(cryptoFeeExchangeAmount))
+  const feeAmountInUSD = convertCurrencyFromExchangeRates(exchangeRates, currencyCode, 'iso:USD', parseFloat(cryptoFeeExchangeAmount))
+  return {
+    amount: bns.toFixed(fiatFeeAmount.toString(), 2, 2),
+    style: feeAmountInUSD > FEE_ALERT_THRESHOLD ? feeStyle.danger : feeAmountInUSD > FEE_COLOR_THRESHOLD ? feeStyle.warning : undefined
+  }
+}
+
+export const calculateTransactionFee = (
```
Comment (amanalili) [inline src/util/utils.js:692]: convertTransactionFeeToDisplayFee

File: src/util/utils.js [hunk 4]
```diff
@@ -658,3 +660,90 @@ export function maxPrimaryCurrencyConversionDecimals(primaryPrecision: number, p
   const newPrimaryPrecision = primaryPrecision - precisionAdjustValue
   return newPrimaryPrecision >= 0 ? newPrimaryPrecision : 0
 }
+
+// Fees calculation
```
Comment (amanalili) [inline src/util/utils.js:664]: Convert Transaction Fee to Display Fee

Comment (amanalili) [reply src/util/utils.js:664]: Write a description of this function

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**